### DISCUSSION TITLE
500 Internal Server Error After login

### DISCUSSION THREAD

**Discussion by kenjs** [Leantime/leantime#1306] [Q&A]
Install v2.3.5 with apache server, after install database and user account can login to dashboard,
but  http://localhost/dashboard/home got 500 error .
![20230117214916](https://user-images.githubusercontent.com/1918055/212918278-048e7407-6300-4892-9222-5f2506bfeafe.png)

**Comment by marcelfolaron** [comment]
Any errors in logs/error.log ?

**Comment by marcelfolaron** [comment]
Please install the php gd extension
